### PR TITLE
Update execution.rs

### PR DESCRIPTION
**Pull request by DethRaid**

### CODE REVIEW

File: bve/src/parse/mesh/instructions/execution.rs [hunk 1]
```diff
@@ -72,6 +73,7 @@ impl Instruction {
     }
 }
 
+// Weird that this function has an output argument while `flat_shading` returns its output. Should be consistent
```
Comment (cwfitzgerald) [inline bve/src/parse/mesh/instructions/execution.rs]: The idea was to.not have to create an intermediate vector but I should probably just use SmallVec and call it a day

Comment (DethRaid) [reply bve/src/parse/mesh/instructions/execution.rs]: I don't think it'd be a problem. This is just loading code

File: bve/src/parse/mesh/instructions/execution.rs [hunk 2]
```diff
@@ -156,6 +158,7 @@ impl Executable for CreateMeshBuilder {
 
             let first: &PolygonFace = group.peek().expect("Groups must not be empty");
             // All of these attributes are the same, so we can just grab them from the first one.
+            // Yay duplicate data
```
Comment (cwfitzgerald) [inline bve/src/parse/mesh/instructions/execution.rs]: I need to go back on a conversation with my mod to see if this is actually duplicate data in the spec or if I'm reading the spec wrong, iirc I was reading the spec wrong and it was impossible for it to be different so I should just store this on a per mesh (as opposed to a pet polygon) basis.

File: bve/src/parse/mesh/instructions/execution.rs [hunk 3]
```diff
@@ -182,7 +185,7 @@ impl Executable for CreateMeshBuilder {
 
             calculate_normals(&mut mesh);
 
-            ctx.pso.meshes.push(mesh);
+            ctx.pso.meshes.push(mesh);  // I keep thinking that PSO means Pipeline State Object
         }
```
Comment (cwfitzgerald) [inline bve/src/parse/mesh/instructions/execution.rs]: My shitty naming conventions are shitty.

File: bve/src/parse/mesh/instructions/execution.rs [hunk 4]
```diff
@@ -369,7 +372,7 @@ impl Executable for Rotate {
     }
 }
 
-impl Executable for Sheer {
+impl Executable for Sheer { // Do you mean shear?
```
Comment (cwfitzgerald) [inline bve/src/parse/mesh/instructions/execution.rs]: Yup :|

File: bve/src/parse/mesh/instructions/execution.rs (hunk 3 above)
Comment (DethRaid) [reply bve/src/parse/mesh/instructions/execution.rs]: F

File: bve/src/parse/mesh/instructions/execution.rs (hunk 1 above)
Comment (cwfitzgerald) [reply bve/src/parse/mesh/instructions/execution.rs]: Yeah you right

File: bve/src/parse/mesh/instructions/execution.rs (hunk 4 above)
Comment (DethRaid) [reply bve/src/parse/mesh/instructions/execution.rs]: F

File: bve/src/parse/mesh/instructions/execution.rs (hunk 1 above)
Comment (cwfitzgerald) [reply bve/src/parse/mesh/instructions/execution.rs]: Solved in 2c6c0d7625440ce3dc0c9da4a1f76c5b83821b6a

File: bve/src/parse/mesh/instructions/execution.rs (hunk 2 above)
Comment (cwfitzgerald) [reply bve/src/parse/mesh/instructions/execution.rs]: Not solved, but going to resolve later on the TODO.

**Pull request closed without merging**